server/grpc_test.go

```diff
@@ -410,6 +412,65 @@ func TestGrpcAcRequestInlinedBlobs(t *testing.T) {
 	}
 }
 
+func TestGrpcByteStreamDeadline(t *testing.T) {
+	testBlobSize := int64(maxChunkSize * 3 / 2)
+	testBlob, testBlobHash := testutils.RandomDataAndHash(testBlobSize)
+	testBlobDigest := pb.Digest{
+		Hash:      testBlobHash,
+		SizeBytes: int64(len(testBlob)),
+	}
+
+	instance := "deadlineExpired"
+
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(10*time.Millisecond))
+	defer cancel()
+
+	bswc, err := bsClient.Write(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	resourceName := fmt.Sprintf("%s/uploads/%s/blobs/%s/%d/deadline/metadata/here", instance, uuid.New().String(), testBlobDigest.Hash, len(testBlob))
+	fmt.Println(testBlobHash)
```

Comment:
This doesn't look particularly useful, should we remove it? If not, please use `t.Log()` instead.

Reply:
done